x/docs/md/cli/mac.md

```diff
@@ -25,7 +25,9 @@ Homebrew/homebrew-core (git revision 3b4c; last commit 2016-05-09)
 Install the CLI via homebrew in the following section.  Otherwise see
 "[Installing Without Homebrew](#installing-without-homebrew)".
 
-### Installing With Homebrew <a name="installing-with-homebrew"></a>
+
+<a name="installing-with-homebrew"></a>
```

Comment:
This is a markdown file.
Markdown has it's own syntax for adding anchors and one should not mix html/markdown formatting.
Since the anchor is the same as the heading, The anchors to the headings should be created by whatever parses the markdown.